resources/views/members/index.blade.php

```diff
@@ -130,43 +135,43 @@
 						<td>{{ $member->telefoon }}</td>
 						<td><a href="mailto:{{ $member->email }}">{{ $member->email }}</a></td>
 					</tr>
-				@endforeach
-			</tbody>
-		</table>
+					@endforeach
+				</tbody>
+			</table>
+		</div>
+		@endcan
 	</div>
-	@endcan
-  </div>
 
 </div>
 @endsection
 
 @section('footer')
 <!-- These scripts load DataTables -->
 <script type="text/javascript">
-$( document ).ready(function() {
+	$( document ).ready(function() {
+
+	// Create the column options for the members table (number of columns can vary with user capabilities)
+	let memberColumnCount = $('#currentMembersTable').find('tr:first th').length;
+	let orderableColumnIndices = [0, 2, 3];
```

Comment:
Betekend dit niet dat je altijd kolom 0, 2 en 3 orderable zijn? Wat als we in de toekomst na een van die conditionele kolomen een orderable willen? Is het een idee om in de  DOM een attribute toe te voegen bv `data-orderable`? Dan kan je met jQuery over de kolomen heen itereren en toevoegen als die attribute erop zit. Dit kan je ophalen met `$(el).data("data-orderable");` of  met `el.getAttribute("data-orderable")`.